Update README.md (#168)
iptv-org/awesome-iptv: README.md

- [World TV Mobile](https://worldtvmobile.com/) - Search for the channel you prefer to tune in and stream in one click.
- [IPTVnator](https://iptvnator.vercel.app/) - Open-source and cross-platform IPTV player application with multiple features, such as support of m3u and m3u8 playlists, favorites, TV archive/catchup and more.
- [VidGrid](https://vidgrid.tk.gg) - News channel focused multiviewer with support for live m3u8 playlists, YouTube and Twitch streams with one-click audio switching.

> - [IPTV SMARTERS](http://webtv.iptvsmarters.com) - IPTV Smarter Player doesn't contain any playlist, You have to add your own playlist provided by your service provider.

Reviewer: ```suggestion
- [IPTV Smarter Player](http://webtv.iptvsmarters.com) - Video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
```

- [nPlayer](https://apps.apple.com/us/app/nplayer/id1116905928) - Media player with support DTS (DTS HD), DTS Headphone:X, Dolby (AC3, E-AC3).
- [FastoTV Lite](https://apps.apple.com/us/app/fastotvlite/id1496936356) - Open source ad-free IPTV client, with support for live media, VOD, favorites list and EPG.
- [WhatsUp TV](https://apps.apple.com/us/app/whatsup-tv/id1476950273) - Player for IPTV (.m3u) playlists.

> - [Smarters Player Lite](https://apps.apple.com/in/app/smarters-player-lite/id1628995509) - The IPTV Smarters App is a video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.

Reviewer: ```suggestion
- [Smarters Player Lite](https://apps.apple.com/in/app/smarters-player-lite/id1628995509) - Video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
```

> - [Smarters Player Lite](https://apps.apple.com/in/app/smarters-player-lite/id1628995509) - The IPTV Smarters App is a fabulous video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.

Reviewer: ```suggestion
- [Smarters Player Lite](https://apps.apple.com/in/app/smarters-player-lite/id1628995509) - The IPTV Smarters App is a video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
```

- [OTT Navigator IPTV](https://play.google.com/store/apps/details?id=studio.scillarium.ottnavigator) - Free IPTV player for any device (phone, tablet, TV, TV-box).
- [DangoPlayer](https://play.google.com/store/apps/details?id=com.aleapps.videotime) - A simple lightweight and fast video player and IPTV client for Android.
- [DangoPlayer TV](https://play.google.com/store/apps/details?id=com.aleapps.videotimetv) - Port of the DangoPlayer IPTV client for Android TV and Google TV.

> - [IPTV Smarters App](https://www.iptvsmarters.com/#downloads) - The IPTV Smarters App is a video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.

Reviewer: ```suggestion
- [IPTV Smarters App](https://www.iptvsmarters.com/#downloads) - Video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
```

> - [IPTV SMARTERS](https://www.iptvsmarters.com/download?download=android) - The IPTV Smarters App is a fabulous video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.

Reviewer: ```suggestion
- [IPTV Smarters App](https://www.iptvsmarters.com/#downloads) - The IPTV Smarters App is a video streaming player that allows your IPTV customers or end-users to stream content like Live TV, VOD, Series, and TV Catchup supplied by you.
```

- [deepepg](https://www.deepepg.com/) - EPG for China.
- [epgshare01/share01](https://github.com/epgshare01/share01) - Program guides for multiple countries.
- [EPG.pw](https://epg.pw/) - EPG service provider for personalized Electronic Program Guide for your IPTV all around the world.

> - [bevy.be](https://www.bevy.be/) - Provides 2 days of EPG for free, sorted by country.

Reviewer: ```suggestion
- [bevy.be](https://www.bevy.be/epg-guide/) - Free EPG for 2 days, sorted by country.
```
